.github/workflows/ci.yml

```diff
@@ -311,8 +311,7 @@ jobs:
         if: always() && steps.finishPrepare.outcome == 'success'
         # Use the --no-fix flag in push builds to get a failed CI status
         run: >
-          npm run lint -- --max-warnings 0 --format stylish
-          ${{ github.event_name != 'pull_request' && '--no-fix' || '' }}
```

Comment:
Why has this line been removed? It's important for generating a failed CI status (see the comment above). Also the `stylish` format is needed for GitHub's automatic problem matchers.

Reply:
It caused the script to fail for me and when I looked into it, I believe I found that `stylish` was deprecated? But this was some weeks ago, so I only remember vaguely. Will try to bring it back and see if it works now.

Reply:
Works 👍